values.yaml

```diff
@@ -0,0 +1,35 @@
+redis:
+  enabled: true
+  auth:
+    enabled: false
+  architecture: standalone
+  master:
+    persistence:
+      enabled: true
+      size: 1Gi  # Adjust the size as needed
+
+onetimesecret:
+  image: 
+    registry: docker.io
+    repository: onetimesecret/onetimesecret
+    tag: latest #will be overwritten by Chart.yaml appVersion
```

Comment:
This value is useless in its current state, because you're using appVersion in any case.
I think it would be better to use `.Values.tag` and default to `.Chart.AppVersion` if its null.

Reply:
added logic for `.Values.tag` and removed it from `values.yaml`
[f73950f](https://github.com/onetimesecret/helm-charts/pull/1/commits/f73950fd7972e539bc209b1499b9f6541abf7308)

Reply:
Great, thanks. I think it's best to leave the default value in the `values.yaml` file, as a lot of user would go and read this value to see the possible configuration options.
You can either leave the value set to null (`tag: ~`) or just keep it commented.

Reply:
I have kept tag commentd...also added note that 'this will override .Chart.AppVersion' .